packages/world/gas-report.json

```diff
@@ -3,43 +3,43 @@
     "file": "test/AccessControl.t.sol",
     "test": "testAccessControl",
     "name": "AccessControl: hasAccess (cold)",
-    "gasUsed": 6521
+    "gasUsed": 9111
```

Comment:
I assume this increased because the tests still use the non-`_`-prefixed method, should we add variants of each of these for the `_` methods to compare and still have a gas report for the internally used methods?

Reply:
I didn't think it was necessary to have gas reports for all the internal `_` method variants (since they're internal) and the gas use is captured by the tests using these internal methods (which didn't update in this PR since there was no gas change)